pennylane/templates/broadcast.py

```diff
@@ -58,10 +59,13 @@ def wires_all_to_all(wires):
     sequence = []
     for i in range(len(wires)):
         for j in range(i + 1, len(wires)):
-            sequence += [[wires[i], wires[j]]]
+            sequence += [wires.subset([i, j])]
     return sequence
 
 
+###################
```

Comment:
Very minor, but would be good to standardise the length of this, e.g., I see a shorter one on line 489. We could do the standard 79 characters or just skip this line altogether.

Reply:
Do you mind if I mark this for the future? You are right, the formatting should be standarised!